coderd/database/dbtestutil/db.go

```diff
@@ -19,9 +23,30 @@ func WillUsePostgres() bool {
 	return os.Getenv("DB") != ""
 }
 
-func NewDB(t testing.TB) (database.Store, pubsub.Pubsub) {
+type options struct {
+	fixedTimezone string
+}
+
+type Option func(*options)
+
+// WithTimezone sets the database to the defined timezone instead of
+// to a random one.
+//
+// Deprecated: If you need to use this, you may have a timezone-related bug.
+func WithTimezone(tz string) Option {
+	return func(o *options) {
+		o.fixedTimezone = tz
+	}
+}
```

Comment:
review: I originally exposed this as `WithFixedTimezone()` but I quickly realized that it's useful to be able to set a specific timezone in unit tests, especially when you're trying to narrow down the root cause of a timezone-related bug.